src/datatype/operators_bvecf32.rs

```diff
@@ -94,3 +95,83 @@ fn _vectors_bvecf32_operator_jaccard(lhs: BVecf32Input<'_>, rhs: BVecf32Input<'_
     check_matched_dims(lhs.dims() as _, rhs.dims() as _);
     BVecf32Jaccard::distance(lhs.for_borrow(), rhs.for_borrow()).to_f32()
 }
+
+#[pgrx::pg_extern(immutable, strict, parallel_safe)]
+fn _vectors_bvecf32_sphere_l2_in(
+    lhs: BVecf32Input<'_>,
+    rhs: pgrx::composite_type!("sphere_bvector"),
+) -> bool {
+    let center: BVecf32Output = match rhs.get_by_index(NonZero::new(1).unwrap()) {
+        Ok(Some(s)) => s,
+        Ok(None) => pgrx::error!("Bad input: empty center at sphere"),
+        Err(e) => pgrx::error!("Parse center failed at sphere:{e}"),
```

Comment:
```suggestion
        Err(e) => unreachable!(),
```

It only occurs if there is no such attribute or it's not of type `BVecf32Output`.